aten/src/ATen/test/basic.cpp

```diff
@@ -471,3 +474,48 @@ TEST(BasicTest, FactoryMethodsTest) {
     ASSERT_FALSE(tensor0.is_pinned());
   }
 }
+
+TEST(BasicTest, BasicStdTestCPU) {
+  c10::once_flag flag1, flag2;
```

Comment:
Note: if we switch this from c10::once_flag to std::once_flag (along witht he call_once down below), this test hangs on my machine, where i have `/usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.28`